bukkit/src/main/java/net/vyhub/command/Ban.java

```diff
@@ -0,0 +1,96 @@
+package net.vyhub.command;
```

Comment:
This class includes more functions than just a commands. We probably should separate that.